README.md

```diff
@@ -11,27 +11,33 @@ MPEG-2 Transport Stream is a very wide spread format for transporting media.
 This repo provides some tools to facilitate analysis and extraction of
 data from MPEG-2 TS streams.
 
-## ts-info
+## mp2ts-info
 
-`ts-info` is a tool that parses a TS file, or a stream on stdin, and prints
+`mp2ts-info` is a tool that parses a TS file, or a stream on stdin, and prints
 information about the video streams in JSON format.
-For AVC (H.264), it shows information about
-PTS/DTS, PicTiming SEI, SPS and PPS, and NAL units.
+
+## mp2ts-pslister
+
+`mp2ts-pslister` is a tool that shows information about SPS, PPS (and VPS in H.265) in a TS file.
```

Comment:
HEVC is a more common name than H.265, so I prefer using it.
One typically says H.264 and HEVC mixing the two ;-)

We may also write HEVC (H.265) for clarity